### PR TITLE
migrate repadmin command

### PR DESCRIPTION
**Pull request by RainbowDashLabs**

### CODE REVIEW

File: src/main/resources/locale_de_DE.properties [hunk 1]
```diff
@@ -367,3 +367,9 @@ words.reputation=Reputation
 words.reputationOffset=Reputationsversatz
 words.role=Rolle
 words.unknown=Unbekannt
+command.repadmin.reputation.add.user.description=Der Nutzer, um Reputation hinzuzufügen.
+command.repadmin.reputation.set.user.description=Der Nutzer, der die Reputation festlegt.
+command.repadmin.reputation.remove.user.description=Der Nutzer, um Reputation zu entfernen.
+command.repadmin.reputation.add.amount.description=Der Betrag zum Hinzufügen.
+command.repadmin.reputation.set.amount.description=Der zu setzende Betrag.
+command.repadmin.reputation.remove.amount.description=Der zu entfernende Betrag.
```
Comment (Taucher2003) [inline src/main/resources/locale_de_DE.properties:375]: These keys are out of order

File: src/main/java/de/chojo/repbot/commands/repadmin/handler/Profile.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import de.chojo.repbot.config.Configuration;
+import de.chojo.repbot.dao.provider.Guilds;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+
+public class Profile implements SlashHandler {
+    private final Guilds guilds;
+    private final Configuration configuration;
+
+    public Profile(Guilds guilds, Configuration configuration) {
+
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/commands/repadmin/handler/Profile.java:14]:
```suggestion
```

File: src/main/resources/locale_de_DE.properties (hunk 1 above)
Comment (RainbowDashLabs) [reply src/main/resources/locale_de_DE.properties:375]: I will order them once I merged it. this is causing too much merge conflicts x.x

Comment (Taucher2003) [reply src/main/resources/locale_de_DE.properties:375]: Seems fine to do that

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**